app/views/reasonableExcuseJourneys/other/noJs/UploadFirstDocumentPage.scala.html

```diff
@@ -65,7 +66,8 @@ <h1 class="govuk-heading-l">@messages("otherReason.uploadEvidence.heading")</h1>
             @fileUploadDialog(form)
         </div>
         <p>
-            @govukButton(Button(content = Text(messages("otherReason.uploadEvidence.noJs.button.uploadFile"))))
+            <a href="@nextPageWithUpload" class="govuk-button">@messages("otherReason.uploadEvidence.noJs.button.uploadFile")</a>
```

Comment:
Hyperlinks do not submit the form, this would just redirect the user and nothing would happen. We will need that button back